guides/common/modules/proc_salt-importing-salt-states.adoc

```diff
@@ -0,0 +1,22 @@
+[id="salt_guide_importing_salt_states_{context}"]
+= Importing Salt States
+
+A Salt State configures parts of a host, for example a service or the installation of a package.
+You can import Salt States from your Salt Master to {Project}.
+The Salt Master configuration used above uses a Salt Environment called `base` which includes the Salt States stored in `/srv/salt/`.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Configure > Salt States* and click *Import from FQDN*.
+** Click *Edit* to assign Salt States to Salt Environments.
+** Click *Delete* to remove a Salt State from your {Project}.
+This only removes the Salt State from {Project}, not from the disk of your Salt Master.
+. Click *Submit* to import the Salt States.
+
+Once imported, you can assign those Salt States to hosts or host groups.
+Salt applies these states to any hosts they are assigned to every time you {ManagingHostsDocURL}#salt_guide_running_salt[run] `state.highstate`.
+
+[NOTE]
+====
+Configure the paths for Salt States and Salt Pillars in `/etc/salt/master`.
+By default, Salt States are located in `/srv/salt` and and Salt Pillars in `/srv/pillar`.
+====
```

Comment:
Warning: NOTE

Reply:
It's a note. Does this need to be changed?

Reply:
No :)